Tests/Serializer/BaseSerializerHandlerTest.php

```diff
@@ -19,7 +19,7 @@
  */
 final class BaseSerializerHandlerTest extends \PHPUnit_Framework_TestCase
 {
-    public function testGetSubscribingMethods()
+    public function testGetSubscribingMethodsWithDefaultFormats()
```

Comment:
Should be added to the `@legacy` group, right ? Don't you get deprecation notices when running that? Also, add a `NEXT_MAJOR` label to say this should call `setFormats`